src/lib/utils.ts

```diff
@@ -174,8 +174,7 @@ export function parsePacket({ type, attributes }: ParsedEvent): Packet {
   if (type !== 'send_packet') {
     throw new Error(`Cannot parse event of type ${type}`);
   }
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
And the root cause was ignoring eslint and using "any" which didn't check the proper types.